### ISSUE TITLE
Django 1.8-1.10 not actually supported

### ISSUE THREAD

**Issue by geoff-va** [juanifioren/django-oidc-provider#345]
It appears Django 1.8-1.10 are not actually supported because `LogoutView` wasn't created until [Django 1.11](https://docs.djangoproject.com/en/1.11/topics/auth/default/#django.contrib.auth.views.LogoutView). So either the docs could be updated or we can add support for that class based view.

I'm not sure what else may/may not work in pre django 1.11 even if that is fixed.

This was discovered by installing the application in a django 1.8 environment and an import error occurring: `ImportError: cannot import name LogoutView`.

**Comment by geoff-va** [comment]
It appears the only things causing the incompatibility are the `LoginView` and `LogoutView` classes that were new in Django 1.11.

* I copied those classes from django 1.11 and added them to the `compat.py` file that already existed
* Updated both `urls.py` files to catch import errors and load the ones from `compat.py` if not found
* Updated `views.py` to do the same thing

I ran tox with python 2.7 and django 1.8.18 and all tests are passing, which is encouraging at least. Still need to run with the remaining environments.
